lib/CapabilityDelegation.js

```diff
@@ -62,6 +67,11 @@ module.exports = class CapabilityDelegation extends ControllerProofPurpose {
       }
     }
     this.suite = suite;
+
+    if(currentDate && !(currentDate instanceof Date)) {
```

Comment:
Docs for this class say date defaults to `new Date()`, but I think that default happens in util.verifyCapabilityChain?  Maybe this check should move into that function?  Now you could pass in null here and it would get through to the comparisons.  Might be able to just do the `instanceof` check.

Reply:
Attempted to address your concerns here: 32e102b746c3b6443e855917c5a9a51b34b83b9d